packages/mgt-components/src/components/mgt-todo/mgt-todo.scss

```diff
@@ -6,12 +6,121 @@
  */
 
 @import '../mgt-tasks-base/mgt-tasks-base.scss';
+@import '../mgt-tasks-base/mgt-tasks-base.theme.scss';
 
 :host {
   @extend :host;
   border-radius: 8px;
   .Task {
     @extend .Task;
-    border: none;
+    width: 800px;
```

Comment:
This fixes the width to 800px and causes the textfield to "overflow" below the JS section in stroybook.

![image](https://user-images.githubusercontent.com/8081536/223698047-ed01f9e5-a1dd-4994-aaf0-60720cf932b0.png)

If you change the width on `.Task` to auto it fixes it:

![image](https://user-images.githubusercontent.com/8081536/223699315-c86710ba-3e3b-407e-b560-79a008853b0b.png)
```suggestion
    width: auto;
```